src/ui/Header/Sidebar/AllTabs/Watch-Later.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import SideTag from "../../../SideTag";
+import "./index.css";
+import { ReactComponent as WatchLaterIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/icons/baricons/watch-later.svg";
+import { ReactComponent as ActiveWatchLaterIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/icons/activebaricons/activeWatchLater.svg";
```

Comment:
this import is incorrect, this is a local file and not within the project folder